### ISSUE TITLE
FP isolator (Does not allow using kip - ft units)

### ISSUE THREAD

**Issue by samyogshr** [idaholab/mastodon#325]
Please check line 54 of ComputeFPIsolatorElasticity.C

**Comment by somu15** [comment]
@samyogshr Thanks for pointing this out. Are you saying that line 54 should be "8 >= unit > 0"?

**Comment by cbolisetti** [comment]
Good catch. Can one of you create a PR for these? @samyogshr if you'd like to learn how to do it, @somu15 or @mriganabh can help. Thanks for pointing these out.
